packages/app/src/app/overmind/effects/vscode/sandboxFsSync/utils.ts

```diff
@@ -0,0 +1,34 @@
+import { Directory, Module, SandboxFs } from '@codesandbox/common/lib/types';
+import { json } from 'overmind';
+
+export const writeFile = (fs: SandboxFs, module: Module) => {
+  fs[module.path] = module;
+};
+
+export const rename = (fs: SandboxFs, fromPath: string, toPath: string) => {
+  Object.keys(fs).forEach(path => {
+    if (path.startsWith(fromPath)) {
```

Comment:
This check wouldn't work in cases like this:

Let's say I rename `/test/a` to `/test/b`. In this case if there's a file called `/test/another.js` it would be renamed as well, to `/test/bnother.js`. We can fix it by changing the check to see if `path.startsWith(query + '/)` or `path.startsWith(query + <end of line>)`.